src/main/webapp/i18n/de/assessment.json

```diff
@@ -13,6 +14,21 @@
                 "overrideAssessment": "Bewertung überschreiben",
                 "nextSubmission": "Nächste Abgabe bewerten",
                 "resolveConflict": "Konflikte behandeln"
+            },
+            "detail": {
+                "feedback": "Feedback",
+                "scoreFor": "Bewertung für"
+            },
+            "messages": {
+                "confirmCancel": "Bist du sicher, dass du die Bewertung abbrechen möchtest? Dein aktueller Bewertungsfortschritt wird gelöscht und die Abgabe steht anderen Tutoren wieder zur Verfügung.",
+                "loadSubmissionFailed": "Laden der angefragten Abgabe ist fehlgeschlagen.",
+                "submitSuccessful": "Deine Bewertung wurde erfolgreich abgesendet!",
+                "lock": "Dieses Modell kann jetzt exklusiv nur noch von dir bearbeitet werden. Bitte bewerte dieses Modell bevor du ein neues öffnest.",
```

Comment:
Is `Modell` the right term here? Isn't the assessment neutral to the exercise type?

Reply:
You are right here. I wanted to unify translations, but didn't see that this is inappropriate for file-upload assessment. Changed this